tools/namefixer.py

```diff
@@ -1346,8 +1347,149 @@
     "TRANS_TYPE_20": "TRANS_TYPE_FADE_DYNAMIC",
     "TRANS_TYPE_21": "TRANS_TYPE_CIRCLE",
     "TRANS_TYPE_22": "TRANS_TYPE_WIPE5",
-    "PLAYER_CSMODE_6": "PLAYER_CSMODE_END",
-    "PLAYER_CSMODE_7": "PLAYER_CSMODE_WAIT",
+    "PLAYER_CSMODE_NONE": "PLAYER_CSACTION_NONE",
+    "PLAYER_CSMODE_1": "PLAYER_CSACTION_1",
+    "PLAYER_CSMODE_2": "PLAYER_CSACTION_2",
+    "PLAYER_CSMODE_3": "PLAYER_CSACTION_3",
+    "PLAYER_CSMODE_4": "PLAYER_CSACTION_4",
+    "PLAYER_CSMODE_5": "PLAYER_CSACTION_5",
+    "PLAYER_CSMODE_6": "PLAYER_CSACTION_END",
+    "PLAYER_CSMODE_END": "PLAYER_CSACTION_END",
+    "PLAYER_CSMODE_7": "PLAYER_CSACTION_WAIT",
+    "PLAYER_CSMODE_WAIT": "PLAYER_CSACTION_WAIT",
+    "PLAYER_CSMODE_8": "PLAYER_CSACTION_8",
+    "PLAYER_CSMODE_9": "PLAYER_CSACTION_9",
+    "PLAYER_CSMODE_10": "PLAYER_CSACTION_10",
+    "PLAYER_CSMODE_11": "PLAYER_CSACTION_11",
+    "PLAYER_CSMODE_12": "PLAYER_CSACTION_12",
+    "PLAYER_CSMODE_13": "PLAYER_CSACTION_13",
+    "PLAYER_CSMODE_14": "PLAYER_CSACTION_14",
```

Comment:
oof. it may be better to add those to the `simpleReplace` dictionary instead

Reply:
Ah, that's so much better, didn't realize that was there. Thanks!